src/services/rfqm_service.ts

```diff
@@ -242,7 +252,17 @@ export class RfqmService {
         );
         const metaTransactionHash = metaTransaction.getHash();
 
-        // TODO: Save a record of the metatransactionHash (indexed), orderHash, and fee (a json blob)
+        // TODO: Save the makerUri and integratorId
+        // Save the RfqmQuote
+        await this._connection.getRepository(RfqmQuoteEntity).insert(
+            new RfqmQuoteEntity({
+                orderHash,
+                metaTransactionHash,
+                chainId: CHAIN_ID.toString(),
```

Comment:
are we going with numeric representation here? this will be '1'. might be worth being descriptive, e.g. 'Ethereum Mainnet'

Reply:
ended up going with a numeric representation. Keeping the convention that ChainId is a number will reduce confusion. Non EVM chains will likely need much more thought across the stack, so will punt on that